src/Sys/InspectSeoHandler.php

```diff
@@ -0,0 +1,176 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\Sys;
+
+use Defuse\Crypto\Crypto;
+use Gdbots\Ncr\Exception\NodeNotFound;
+use Gdbots\Ncr\Ncr;
+use Gdbots\Pbj\Message;
+use Gdbots\Pbjx\CommandHandler;
+use Gdbots\Pbjx\Pbjx;
+use Gdbots\Schemas\Pbjx\StreamId;
+use Gdbots\UriTemplate\UriTemplateService;
+use Google\Service\SearchConsole\InspectUrlIndexResponse;
+use Psr\Log\LoggerInterface;
+use Triniti\Schemas\Sys\Event\SeoInspectedV1;
+use Defuse\Crypto\Key;
+
+
+final class InspectSeoHandler implements CommandHandler
+{
+    public static function handlesCuries(): array
+    {
+        return [
+            'triniti:sys:command:inspect-seo',
+        ];
+    }
+
+    public function __construct(
+        private readonly Ncr        $ncr,
+        private readonly Key        $key,
+        private readonly Flags      $flags,
+        private readonly LoggerInterface $logger,
+    )
+    {
+    }
+
+    public function handleCommand(Message $command, Pbjx $pbjx): void
+    {
+        if ($this->flags->getBoolean('inspect_seo_handler_disabled')) {
+            return;
+        }
+
+        $searchEngines = $command->get('search_engines', ['google']);
```

Comment:
@efjacobson @digitalartcrew i'd leave off the default here.  if it wasn't populated as a default in the schema or sent with the command we shouldn't assume.  just return if there's no value populated.  defaults in schemas, generally ok, in code, trickier.

Reply:
@gdbrown this was just modeled after a couple site-level `CollectArticleStatsHandler`s that have logic like 


```php
    private function fromGoogle(Message $command, Message $event, Message $article): void
    {
        if ($this->flags->getBoolean('stats_collector_google_disabled')) {
            return;
        }

        if ($command->has('providers') && !$command->isInSet('providers', 'google')) {
            return;
        }
        
        ...
```

which is another way of implementing the (described) [schema](https://github.com/triniti/schemas/blob/ff1a23c3595b2bb10bfb5ffba7551796d7b0058b/schemas/triniti/sys/command/inspect-seo/latest.xml#L12)

the thought was that at this level each checkIndexForBlah would be represented in this default, but i see now that is a bit tricky because there isn't a clean way to override that without overriding the entire `handleCommand`. we could have a dedicated method for it though, something like 

```php
    protected static function getAllSearchEngines(): array
    {
        return ['google'];
    }
```

Reply:
@efjacobson difference is the collect article stats is that the entire handler is site level and deals with the default case when no providers are populated.  you could simply set the "default" when you create the command since that is happening at site level.  then you'd never have a scenario where you'd have no search engines set or if it happened, just ignore it